osu.Game/Screens/Edit/Verify/IssueList.cs

```diff
@@ -83,10 +83,10 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            Refresh();
+            verify.HiddenIssueTypes.BindCollectionChanged((o, s) => refresh(), runOnceImmediately: true);
```

Comment:
```suggestion
            verify.HiddenIssueTypes.BindCollectionChanged((o, s) => refresh(), true);
```

just to match how we do it everywhere else.